hdfs3/_pyarrow.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import
+
+import pyarrow as pa
```

Comment:
Does there need to be a deprecated import here too, like with distributed.hdfs, below?

Reply:
No, this is loaded by `dask.py` only. New functionality.